src/components/Brewery.js

```diff
@@ -42,6 +61,7 @@ export default class Brewery extends Component {
               {city}, {state}
             </h6>
             {this.displayWebsiteUrl(websiteUrl)}
+            {this.displayLocation(street, city, state)}
           </div>
```

Comment:
The argument for `this.displayLocation(...)` would then be `this.displayLocation([street, city, state])` as mentioned in the comment above.

After these changes and running `npm run lint` you should have one warning and one error from eslint. You can dismiss those as they are or will be addressed in different PR.

Reply:
Thank you for these tips! I have made the mentioned changes. I know this does not totally fix #9, but I thought I would at least assist in getting some sort of map feature up for time being.